server/src/main/java/com/google/musicanalysis/site/YoutubeGenres.java

```diff
@@ -68,12 +74,17 @@ protected void calculateMusicCount(JsonArray videos) {
             }
         }
 
-        if (isMusic && totalSubgenres == 0) {
+        if (isMusic) {
+          // likedMusicHistory records video numbers that are music
+          likedMusicHistory.add(firstVideoCount + i);
```

Comment:
Took me a minute to see what you're doing, but now it makes sense. One thing that I noticed tho is that you're passing in a 0 for `firstVideoCount` in your `YoutubeServlet.java` file, which to me makes it useless since you're just adding by zero every time. Why are you passing it through this function? Will there be a case where you pass in a number other than 0?

If you want to keep track of the number of videos retrieved, you could just do
 ```
int videosRetrieved = 0;
...
videosRetrieved += genreAnalysis.calculateMusicCount(videos);
```
since it looks like you're just returning the size of the `videos` jsonArray. 

You also never modify the `videos` jsonArray, so what's the point of returning the size with this function? I think leaving this as a `void` function was more clear.

Reply:
got it, I'll make these changes